sci-geosciences/GeographicLib/GeographicLib-1.51-r1.ebuild

```diff
@@ -0,0 +1,134 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+
+inherit cmake distutils-r1
+
+MY_PN="${PN,,}"
+
+DESCRIPTION="C++ library for converting geographic coordinate systems"
+HOMEPAGE="https://sourceforge.net/projects/geographiclib/"
+SRC_URI="mirror://sourceforge/${MY_PN}/${P}.tar.gz"
+LICENSE="MIT"
+
+# To-Do:find out why this doesn't work
+# 105/167 Test #105: GeodSolve82 ......................***Failed
+# Required regular expression not found. Regex=[20\.0010* 0\.0* 180\.0* 20\.0010* 0\.0* 180\.0* 0\.0* 0\.0* 0\.0* 1\.0* 1\.0* -?0
+RESTRICT="test"
+
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE_PRECISION="
+	precision-arbitrary
+	+precision-double
+	precision-long-double
+	precision-quad
+	precision-single
```

Comment:
I really don't like these useflags: they appear to change the ABI, for example some Math.hpp classes will have different signatures

please run an abi checker tool before and after toggling one of those flags

if they differ, I'm afraid we'll have to pick one precision and stick with it (ideally one that matches other distros so that we can have some sort of binary compatibility)

Reply:
I've remove the precision useflags